crabs/bboxes_labelling/extract_frames_to_label_w_sleap.py

```diff
@@ -325,12 +325,10 @@ def extract_frames_to_label_from_video(
                 raise KeyError(msg)
 
             # If read successfully: save to file
-            # file naming format: parentdir_videoname_frame_XXX.png
+            # file naming format: videoname_frame_XXX.png
             else:
                 file_path = video_output_dir / Path(
-                    f"{Path(vid_str).parent.stem}_"
-                    f"{Path(vid_str).stem}_"
-                    f"frame_{frame_idx:06d}.png",
+                    f"{Path(vid_str).stem}_" f"frame_{frame_idx:06d}.png",
```

Comment:
How scalable is the 6 digits in frame number? Worth a couple more leading zeroes in there for safety? Or not needed.

Reply:
for now all videos have less than 999999 frames (max is 345048 frames), but will had a couple of leading 0s for safety 🧷